util/session/sessionmanager.go

```diff
@@ -73,14 +73,12 @@ func NewSessionManager(settingsMgr *settings.SettingsManager, dexServerAddr stri
 		s.client.Transport = httputil.DebugTransport{T: s.client.Transport}
 	}
 
-	s.disableAdmin = disableAdmin
-
 	return &s
 }
 
 // Create creates a new token for a given subject (user) and returns it as a string.
 // Passing a value of `0` for secondsBeforeExpiry creates a token that never expires.
-func (mgr *SessionManager) Create(subject string, secondsBeforeExpiry int64) (string, error) {
+func (mgr *SessionManager) Create(subject string, secondsBeforeExpiry int64, id string) (string, error) {
```

Comment:
We should clarify what `id` is used for here.

Reply:
done. Updated comment